src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -61,49 +73,41 @@ public Shooter() {
   public void configure() {
     // -- Left Motor --
     leftConfig.MotorOutput.Inverted = constShooter.LEFT_INVERT;
-    leftConfig.Slot0.kV = prefShooter.leftShooterV;
-    leftConfig.Slot0.kS = prefShooter.leftShooterS;
-    leftConfig.Slot0.kA = prefShooter.leftShooterA;
-    leftConfig.Slot0.kP = prefShooter.leftShooterP;
-    leftConfig.Slot0.kI = prefShooter.leftShooterI;
-    leftConfig.Slot0.kD = prefShooter.leftShooterD;
-
-    leftConfig.MotionMagic.MotionMagicAcceleration = 400;
-    leftConfig.MotionMagic.MotionMagicJerk = 4000;
+    leftConfig.Slot0 = constShooter.LEFT_PID_SLOT_0_FAST;
+    leftConfig.Slot1 = constShooter.LEFT_PID_SLOT_1_SLOW;
+
+    leftConfig.MotionMagic.MotionMagicCruiseVelocity = 60;
+    leftConfig.MotionMagic.MotionMagicAcceleration = 600;
+    leftConfig.MotionMagic.MotionMagicJerk = 6000;
```

Comment:
How does motion magic work with slots? Can we do motion magic on a slot

Reply:
Motion magic doesn't accept a slot as a parameter. It's settings are stored to the entire motor.
We can determine if we want to use motion magic when we create the request, as we need to create a `MotionMagicVoltage` object to use it. If you wanted to only use motion magic on a certain slot, you could just use a different request type whenever using that slot